ingester/ingester.go

```diff
@@ -547,7 +478,7 @@ func (i *Ingester) sweepSeries(userID string, fp model.Fingerprint, series *memo
 		return
 	}
 
-	lastTime := series.lastTime
+	lastTime := series.firstTime()
```

Comment:
haha yes I should rename the variable.  This is the change for "use series's first time as the sort key for the flush queue, not last time".